src/Core/Retry/CommandRetry.php

```diff
@@ -60,15 +60,15 @@ public function __construct(RetryStrategyInterface $strategy, int $maxRetries =
      *
      * @param callable $action The callable action to execute with a retry strategy
      * @return mixed The return value of the passed action callable
-     * @throws \Exception
+     * @throws \Throwable
```

Comment:
Updated to use Throwable and psalm still thinks there is another return path

Reply:
It probably can't figure out all the code paths due to the `while(true)`. We can just ignore the error.